pkg/netpol/diff/diff.go

```diff
@@ -215,28 +310,97 @@ func getKeyFromP2PConn(c connlist.Peer2PeerConnection) string {
 	return src.String() + keyElemSep + dst.String()
 }
 
+type DiffTypeStr string
```

Comment:
Maybe move the definition of this enum to `connectivity_diff.go` as this is part of the declared interface

Reply:
done